src/main/java/com/epam/aidial/core/function/CollectModelStatsFn.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.aidial.core.function;
+
+import com.epam.aidial.core.Proxy;
+import com.epam.aidial.core.ProxyContext;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+public class CollectModelStatsFn extends BaseFunction<ObjectNode> {
+    public CollectModelStatsFn(Proxy proxy, ProxyContext context) {
+        super(proxy, context);
+    }
+
+    @Override
+    public Throwable apply(ObjectNode requestBody) {
```

Comment:
please add a javadoc or comment with description why we do not catch error from this code and return null in any case

Reply:
fixed